src/pybind/mgr/dashboard/frontend/src/app/ceph/cephfs/cephfs-mount-details/cephfs-mount-details.component.ts

```diff
@@ -10,23 +14,77 @@ import { BaseModal } from 'carbon-components-angular';
 export class CephfsMountDetailsComponent extends BaseModal implements OnInit {
   @ViewChild('mountDetailsTpl', { static: true })
   mountDetailsTpl: any;
+  mountDetail!: Record<string, any>;
   onCancel?: Function;
   private MOUNT_DIRECTORY = '<MOUNT_DIRECTORY>';
-  mountData!: Record<string, any>;
-  constructor(public activeModal: NgbActiveModal) {
-    super();
-  }
+  filteredUsers: any[] = [];
+  selectedUser: string = 'client.admin';
+  updatedUser: any;
+  publicaddr: any;
+  secret: any;
+  userForm: CdFormGroup;
   mount!: string;
+  cephfsmount!: string;
   fuse!: string;
   nfs!: string;
+  extractedKey!: string;
+
+  constructor(
+    public activeModal: NgbActiveModal,
+    private monitorService: MonitorService,
+    public cephUserService: CephUserService,
+    @Optional() @Inject('mountData') public mountData: object[]
```

Comment:
Do we still need this, I see we removed these [below](https://github.com/ceph/ceph/pull/60536/files#diff-470f5d97a0231731ea02ef0b6bfcd9ec812d372a5080da2bf2b9133d19f9ee42R215)

Reply:
Removed unused dependency.